app/src/main/java/sensors_in_paradise/sonar/screen_train/TrainingScreen.kt

```diff
@@ -0,0 +1,156 @@
+package sensors_in_paradise.sonar.screen_train
+
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.content.Context
+import android.graphics.Color
+import android.graphics.Typeface
+import android.util.TypedValue
+import android.widget.Button
+import androidx.annotation.AttrRes
+import androidx.annotation.ColorInt
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.RecyclerView
+import com.github.mikephil.charting.animation.Easing
+import com.github.mikephil.charting.charts.PieChart
+import com.github.mikephil.charting.data.PieData
+import com.github.mikephil.charting.data.PieDataSet
+import com.github.mikephil.charting.data.PieEntry
+import com.github.mikephil.charting.formatter.ValueFormatter
+import com.github.mikephil.charting.utils.ColorTemplate
+import com.github.mikephil.charting.utils.MPPointF
+import sensors_in_paradise.sonar.GlobalValues
+import sensors_in_paradise.sonar.R
+import sensors_in_paradise.sonar.ScreenInterface
+import sensors_in_paradise.sonar.screen_recording.RecordingDataManager
+import sensors_in_paradise.sonar.use_cases.UseCase
+
+class TrainingScreen(
```

Comment:
To maybe make data representations visible in other parts of the app some functions here should be moved out of the screen and accessible from other files too